.circleci/config.yml

```
@@ -58,7 +58,7 @@ jobs:
           command: |
             # remove the cache index
             mamba clean -i -y
-            mamba install python=3.10 julia r-base rpy2 \
+            mamba install python=3.10 "julia<=1.10.0" r-base rpy2 \
                           numpy cython -yq
             pip install --upgrade "setuptools<58.5"
             pip install --upgrade --progress-bar off julia
```

Sorry, my bad, actually the example only compares R and python as the name suggests it...
https://github.com/benchopt/benchopt/blob/main/examples/plot_run_benchmark_python_R.py

So we can remove julia from CircleCI altogether.

```suggestion
            mamba install python=3.10 r-base rpy2 \
                          numpy cython -yq
            pip install --upgrade "setuptools<58.5"
```